README.md

```diff
@@ -23,7 +23,38 @@ If you wish to have the server restart on file change, run the server with the `
 `./run_dev_server.sh`
 
 
-##Go say hello
+#### Go say hello
 
 Visit `localhost:3000` to see the hello world.
 
+
+## API Specification
+
+#### Login Methods
+
+TODO - I'm not sure how oauth with a frontend framework will work.
```

Comment:
@karrde00 - thoughts?  Will it be the same as normal oauth, then we'll pass a token to the frontend?